Functions.Templates/PortalTemplates.nuspec

```diff
@@ -92,5 +92,20 @@
     <file src="templates/SendGrid-JavaScript/metadata.json" target="templates/SendGrid-JavaScript/metadata.json" />
     <file src="templates/SendGrid-JavaScript/index.js" target="templates/SendGrid-JavaScript/index.js" />
     <file src="templates/SendGrid-JavaScript/sample.dat" target="templates/SendGrid-JavaScript/sample.dat" />
+    <file src="Templates/BlobTrigger-Java/BlobTriggerJava.java" target="Templates/BlobTrigger-Java/BlobTriggerJava.java" />
+    <file src="Templates/BlobTrigger-Java/metadata.json" target="Templates/BlobTrigger-Java/metadata.json" />
+    <file src="Templates/BlobTrigger-Java/readme.md" target="Templates/BlobTrigger-Java/readme.md" />
+    <file src="Templates/BlobTrigger-Java/sample.dat" target="Templates/BlobTrigger-Java/sample.dat" />
+    <file src="Templates/HttpTrigger-Java/HttpTriggerJava.java" target="Templates/HttpTrigger-Java/HttpTriggerJava.java" />
+    <file src="Templates/HttpTrigger-Java/metadata.json" target="Templates/HttpTrigger-Java/metadata.json" />
+    <file src="Templates/HttpTrigger-Java/sample.dat" target="Templates/HttpTrigger-Java/sample.dat" />
+    <file src="Templates/QueueTrigger-Java/QueueTriggerJava.java" target="Templates/QueueTrigger-Java/QueueTriggerJava.java" />
+    <file src="Templates/QueueTrigger-Java/metadata.json" target="Templates/QueueTrigger-Java/metadata.json" />
+    <file src="Templates/QueueTrigger-Java/readme.md" target="Templates/QueueTrigger-Java/readme.md" />
+    <file src="Templates/QueueTrigger-Java/sample.dat" target="Templates/QueueTrigger-Java/sample.dat" />
+    <file src="Templates/TimerTrigger-Java/TimerTriggerJava.java" target="Templates/TimerTrigger-Java/TimerTriggerJava.java" />
+    <file src="Templates/TimerTrigger-Java/metadata.json" target="Templates/TimerTrigger-Java/metadata.json" />
+    <file src="Templates/TimerTrigger-Java/readme.md" target="Templates/TimerTrigger-Java/readme.md" />
+    <file src="Templates/TimerTrigger-Java/sample.dat" target="Templates/TimerTrigger-Java/sample.dat" />

```

Comment:
Please undo all the changes to this nuspec file. This file contains all the templates packaged for portal.